SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -166,58 +163,12 @@ void USpatialSender::SendAddComponents(Worker_EntityId EntityId, TArray<FWorkerC
 		return;
 	}
 
-	// Update ComponentPresence.
-	check(StaticComponentView->HasAuthority(EntityId, SpatialConstants::COMPONENT_PRESENCE_COMPONENT_ID));
-	ComponentPresence* Presence = StaticComponentView->GetComponentData<ComponentPresence>(EntityId);
-	Presence->AddComponentDataIds(ComponentDatas);
-	FWorkerComponentUpdate Update = Presence->CreateComponentPresenceUpdate();
-	Connection->SendComponentUpdate(EntityId, &Update);
-
-	// Short circuit an enforcer update if possible.
-	NetDriver->LoadBalanceEnforcer->ShortCircuitMaybeRefreshAuthorityDelegation(EntityId);
-
 	for (FWorkerComponentData& ComponentData : ComponentDatas)
 	{
 		Connection->SendAddComponent(EntityId, &ComponentData);
 	}
 }
 
-void USpatialSender::GainAuthorityThenAddComponent(USpatialActorChannel* Channel, UObject* Object, const FClassInfo* Info)
-{
-	Worker_EntityId EntityId = Channel->GetEntityId();
-
-	TSharedRef<FPendingSubobjectAttachment> PendingSubobjectAttachment = MakeShared<FPendingSubobjectAttachment>();
```

Comment:
Don't believe we need `FPendingSubobjectAttachment` anymore, or any related code

Reply:
Agreed. I think we can delete the type and the PendingEntitySubobjectDelegations in spatial receiver too.